python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -530,6 +527,11 @@ def __init__(self, max_tasks_in_flight: int = DEFAULT_MAX_TASKS_IN_FLIGHT):
         self._max_tasks_in_flight = max_tasks_in_flight
         # Number of tasks in flight per actor.
         self._num_tasks_in_flight: Dict[ray.actor.ActorHandle, int] = {}
+        # The maximum number of tasks in flight each actor has run at any point in
+        # time.
+        self._max_tasks_in_flight_per_actor: dict[
```

Comment:
Is it safe to keep actor handle references around? I think they are cleaned up in the dict above, but not here.

Reply:
yeah good point. is there a way to get some sort of id from the actor handle?

Reply:
updated to use `str(actor_handle)` instead of the actor handle itself